dashboard/app/views/levels/_teacher_panel.html.haml

```diff
@@ -7,7 +7,12 @@
 - if @level && @script_level
   - if @level.try(:examples).present? && current_user.authorized_teacher? # 'solutions' for applab-type levels
     - level_example_links = @level.examples.map do |example|
-      - send("#{@level.game.app}_project_view_projects_url".to_sym, channel_id: example, host: 'studio.code.org', port: 443, protocol: :https)
+      // We treat Sprite Lab levels as a sub-set of game lab levels right now which breaks their examples solutions
+      // as level.game.app gets "gamelab" which makes the examples for sprite lab try to open in game lab
+      - if @level.is_a?(GamelabJr)
```

Comment:
Not for this PR, but do we have a plan to rename GamelabJr -> SpriteLab or is that more trouble than it's worth?

Reply:
I think we need to coordinate that with Star Labs probably?